.pre-commit-config.yaml

```
@@ -0,0 +1,138 @@
+    rev: v4.5.0
+    hooks:
+      # Basic file checks
+      - id: trailing-whitespace
+      - id: end-of-file-fixer
+      - id: check-yaml
+      - id: check-json
+      - id: check-toml
+      - id: check-merge-conflict
+      - id: check-case-conflict
+      - id: check-docstring-first
+      
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)